tests/allotrope/schema_parser/schema_cleaner_test.py

```diff
@@ -1072,12 +700,16 @@ def test_combine_allof_with_nested_anyof() -> None:
     validate_cleaned_schema(
         schema,
         {
-            "properties": {
```

Comment:
NOTE to reviewer: these tests change because we don't clean anyof as much anymore.